### PR TITLE
Pl 5

### PR DESCRIPTION
**Pull request by antipixel1**
https://ingesoftg001.atlassian.net/browse/PL-5

### CODE REVIEW

File: pages/api/deleteProject.js [hunk 1]
```diff
@@ -21,9 +27,31 @@ async function deleteProject (req, res) {
...
+        SELECT email FROM hace_uso INNER JOIN esContratado ON esContratado.cedulaEmpleado = hace_uso.cedula
+        AND esContratado.nombreProyecto = ${projectName}
+        AND esContratado.cedulaJuridica = ${companyID};`;
+
+        return employeesEmail;
+    } catch (error) {
+        res.status(500);
+        res.send(error.message);
+    }
+}
```
Comment (julian-solisaguilar) [inline pages/api/deleteProject.js:57]: Add EOF

File: pages/api/deleteProject.js [hunk 2]
```diff
@@ -21,9 +27,31 @@ async function deleteProject (req, res) {
...
         res.status(200).json(updateProject);
     } catch (error) {
         res.status(500);
         res.send(error.message);
     }
 };
+
+async function getEmails(res, companyID, projectName) {
+    try {
+        console.log("llegue a getEmails");
```
Comment (julian-solisaguilar) [inline pages/api/deleteProject.js:46]: Remove debugging logs

File: pages/api/deleteProject.js [hunk 3]
```diff
@@ -21,9 +27,31 @@ async function deleteProject (req, res) {
                 habilitado: false,
             },
         });
+        for (let i = 0; i < employeesEmail.length; i++) {
+            console.log(employeesEmail[i].email);
```
Comment (julian-solisaguilar) [inline pages/api/deleteProject.js:31]: Remove debugging logs

**Review by julian-solisaguilar — CHANGES_REQUESTED** [review]
Add these little changes and we`re good to go. Great work!

**Review by daar2510 — APPROVED** [review]
🚀

**Review by julian-solisaguilar — APPROVED** [review]
LGTM 🚀

**Pull request merged**